productivity-suite/app/legacy-app/src/components/pages/UnimplementedDemoPage.tsx

```diff
@@ -18,8 +19,10 @@ export function UnimplementedDemoPage({
         <p>This page is not implemented as it is out of scope for this demo.</p>
         <p>
           To Interact with SSR fragments view the login form by{" "}
-          <button onClick={auth.logout}>logging out</button> or visit the{" "}
-          <Link to="/todos">todos page</Link>.
+          <button onClick={() => getBus().dispatch("logout", null)}>
```

Comment:
should we consider making the second argument of `dispatch` optional for such type of events?

Reply:
I like the sound of that

Reply:
Yeah I was wondering about that too.
I think that is OK but in such cases what happens when we are serializing the event? We currently serialize to JSON the map of event names to values, but I don't think we can serialize `{ myEvent: undefined }` without losing the event name altogether.

Reply:
what I was thinking was to actually have the second argument optional and it could default to `null`:
```ts
  dispatch(eventName: string, value: JSONValue = null) 
```
so it wouldn't really change anything, just provide a shorthand to dispatch a `null` value

Reply:
PS: I just noticed that we've got an `any` there instead of `JSONValue` 😅 

![Screenshot 2022-11-11 at 21 32 07](https://user-images.githubusercontent.com/61631103/201433846-6bda45cc-68aa-4ad4-bab8-4e1257ca1085.png)